Makefile

```diff
@@ -171,19 +198,19 @@ release-directly-to-prod: dev-env pydist twine-prod
 	@echo "    make install-release"
 
 .PHONY: dev-env
-dev-env: dev-env-client dev-env-local-server
+dev-env: dev-env-client dev-env-server
 
 .PHONY: dev-env-client
 dev-env-client:
 	cd client && $(MAKE) ci
 
-.PHONY: dev-env-local-server
-dev-env-local-server:
-	pip install -r local_server/requirements-dev.txt
-
 .PHONY: dev-env-server
 dev-env-server:
-	pip install -r server/requirements-dev.txt
+	pip install -r backend/server/requirements-dev.txt
+
+.PHONY: dev-env-czi-hosted
+dev-env-czi-hosted:
+	pip3 install -r backend/czi_hosted/requirements-dev.txt
```

Comment:
Were the changes to pip3 intentional? We should probably be uniform as to whether `pip` or `pip3` across this Makefile.

Reply:
I'll revert back to pip